client/src/app/components/home/home.component.html

```diff
@@ -1,2 +1,18 @@
-<app-dashboard></app-dashboard>
-<button (click)="logout()">Logout</button>
```

Comment:
Why did you delete the logout button? Keep it for test purposes until we add it inside the toolbar. Or just create the toolbar using angular material and add it in it.

Reply:
yes i was going to add it to the toolbar

Reply:
perfect.